pipeline/metadata/6.0.yaml

```diff
@@ -791,6 +791,36 @@ pipelines:
             ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
           metadata:
             - rgw
+        - name: "Tier-2 RADOS test clay ecpool"
+          execution_time: ""
+          suite: "suites/quincy/rados/tier-2_rados_test-clay-ecpool.yaml"
+          global-conf: "conf/quincy/rados/11-node-cluster.yaml"
+          platform: "rhel-9"
+          inventory:
+            openstack: "conf/inventory/rhel-9-latest.yaml"
+            ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
+          metadata:
+            - rados
+        - name: "Tier-4 RADOS tests"
+          execution_time: ""
+          suite: "suites/quincy/rados/tier-4_rados_tests.yaml"
+          global-conf: "conf/quincy/rados/11-node-cluster.yaml"
+          platform: "rhel-9"
+          inventory:
+            openstack: "conf/inventory/rhel-9-latest.yaml"
+            ibmc: "conf/inventory/ibm-vpc-rhel-9-latest.yaml"
+          metadata:
+            - rados
+        - name: "Tier-4 RADOS test pool osd recovery"
+          execution_time: ""
+          suite: "suites/quincy/rados/tier-4_rados_test-pool-osd-recovery.yaml"
+          global-conf: "conf/quincy/rados/11-node-cluster.yaml"
```

Comment:
The suite needs to be changed to : 13-node-cluster.yaml , as this requires node replacements.

Reply:
sure, done.